app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.kt

```diff
@@ -48,3 +48,11 @@ fun setImageDrawable(imageView: ImageView, thumbnailGraphic: LessonThumbnailGrap
     }
   )
 }
+
+@BindingAdapter("profile:src")
+fun setProfileImage(imageView: ImageView, imageUrl: String) {
```

Comment:
check function at top of this class `setImageDrawable`, I think that can be used here.

Reply:
Don't we need this for URL loading, though?